src/biogeophys/SoilStateInitTimeConstMod.F90

```diff
@@ -770,7 +781,7 @@ real(r8) function ThresholdSoilMoistZender2003( clay )
          call endrun( 'Clay fraction is out of bounds (0 to 100)')
          return
       end if
-      ThresholdSoilMoistZender2003 = 0.17_r8 + 0.14_r8 * clay * 0.01_r8
```

Comment:
@dmleung is this change important for something? It looks to me that it shouldn't matter...

Reply:
No it doesn't matter. I just deleted the moisture tuning parameter and tidied up the code by grouping the %clay and the 0.01 inside a bracket.